src/main/java/com/jcabi/http/mock/MkAnswerMatchers.java

```diff
@@ -55,10 +54,19 @@ private MkAnswerMatchers() {
     }
 
     /**
-     * Matches the content of {@link MkAnswer#header()} against the given
+     * Matches the value of {@link MkAnswer#content()} against the given
+     *  matcher.
+     * @param matcher The matcher to use.
+     * @return Matcher for checking the body of MkAnswer
+     */
+    public static Matcher<MkAnswer> hasContent(final Matcher<byte[]> matcher) {
+        return new MkAnswerContentMatcher(matcher);
+    }
+
+    /**
+     * Matches the content of {@link MkAnswer#headers()} against the given
```

Comment:
@mkordas because that's correct. It was incorrect before, intelli-J highlighted it.